web/src/routes/admin/(resources)/students/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import DataTable, { type DataItem } from '$lib/elements/ui/dataTable/data-table.svelte';
+  import { deleteStudent, getStudents, type Pageable, type PageStudent } from '$lib/sdk/fetch-client';
+  import { error } from '@sveltejs/kit';
+  import { onMount } from 'svelte';
+  import { writable, type Writable } from 'svelte/store';
+
+  let columnNames = ['Vorname', 'Nachname', 'Tags'];
+  let keys = ['id', 'firstName', 'lastName', 'tags'];
+  let tableData: Writable<DataItem[]> = writable();
+  let totalElements: Writable<number> = writable(0);
+
+  onMount(async () => loadPage(0, '', ''));
+
+  async function loadPage(index: number, sortString: string, filter: string) {
+    let pageable: Pageable = { page: index, size: 10, sort: [sortString] };
+    try {
+      const result: PageStudent = await getStudents(pageable, {
+        firstName: filter,
+        lastName: filter,
+      });
+      totalElements.set(result.totalElements as number);
+      tableData.set(result.content as unknown as DataItem[]);
```

Comment:
Why do you have to cast this to unknown first? I'd expect `StudentResponseDto` and `DataItem` to have sufficient overlap. If not you should really just write a mapping function instead of doing this